### PR TITLE
add created function & made changed index.html.erb

### PR DESCRIPTION
**Pull request by sirinapha88**
Created a function in quotes_controller.rb and made some changed in index.html.erb

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -84,7 +84,7 @@ GEM
       activesupport (= 4.1.1)
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
-    rake (10.3.2)
+    rake (10.5.0)
```
Comment (tebba-von-mathenstein) [inline Gemfile.lock]: Hey, we're actually locked on version 10.3.2 of rake for `reasons`. Please revert this change!

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 module ApplicationHelper
+
```
Comment (tebba-von-mathenstein) [inline app/helpers/application_helper.rb:2]: We don't need the newline, lets keep the git history clean. :) please remove this line.

File: app/views/quotes/index.html.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+        <%= time_ago_in_words(quote.created_at || Time.now) %> ago
```
Comment (tebba-von-mathenstein) [inline app/views/quotes/index.html.erb:12]: This feels like a workaround, are there entries in the database that have a null entry?

Comment (tebba-von-mathenstein) [inline app/views/quotes/index.html.erb:12]: Tum said: Oh yeah theres definitely a null entry in our DB. What should I do?!

Comment (tebba-von-mathenstein) [inline app/views/quotes/index.html.erb:12]: You should find the null entry and uptate it to have a date. You can just use todays date -- something is better than nothing. 

Also, could you look into adding a constraint on the quotes `active_record` model so that we can't find ourselves in this position again?

**Comment by tebba-von-mathenstein** [comment]
Hey Tum, thanks a lot for this work -- I;ve left some comments for you. If you have any questions feel free to ask, and once you've updated the DB model and the DB entries to not have null `created_at` timestamps, I'll finish this review.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (sirinapha88) [inline app/helpers/application_helper.rb:2]: Clean up new line.

**Comment by tebba-von-mathenstein** [comment]
LGTM, I'll handle the merge. Thanks Tum -- you should have heard from Shu on the internal tools team, double check your email if you have not heard anything from Shu yet.
